connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -67,6 +67,7 @@
     protected static final String SYNC_TOPIC_ACLS = "sync.topic.acls";
     protected static final String EMIT_HEARTBEATS = "emit.heartbeats";
     protected static final String EMIT_CHECKPOINTS = "emit.checkpoints";
+    protected static final String SYNC_CONSUMER_OFFSETS = "sync.consumer.offsets";
```

Comment:
elsewhere in MM2 we refer to "groups" but not "consumers". Should this be sync.group.offsets?

Reply:
totally agree